### PR TITLE
hypernova

### PR DESCRIPTION
**Pull request by nao215**
## How to use

npm watch
-> including build js and server side rendering

### CODE REVIEW

File: config/initializers/hypernova.rb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+require 'hypernova'
+
+Hypernova.configure do |config|
+  config.host = 'localhost'
```
Comment (mizutamazukki) [inline config/initializers/hypernova.rb:4]: Go out config host,port
use https://github.com/railsconfig/config
or
make fileloader homebrew

Comment (nao215) [reply config/initializers/hypernova.rb:4]: Nice! Added new issue #19

Comment (mizutamazukki) [reply config/initializers/hypernova.rb:4]: thanks

File: app/controllers/posts_controller.rb [hunk 2]
```diff
@@ -1,6 +1,8 @@
 class PostsController < ApplicationController
   before_action :set_post, only: [:show, :edit, :update, :destroy]
 
+  around_filter :hypernova_render_support
```
Comment (mizutamazukki) [inline app/controllers/posts_controller.rb:4]: Changes to the application controller?

Comment (nao215) [reply app/controllers/posts_controller.rb:4]: Change to application controller will cause unnecessary server side rendering (ex. admin dashboard). 🍣

**Comment by nao215** [comment]
Can I merge this??

**Pull request merged**